Makefile

```diff
@@ -12,11 +12,15 @@ SIDECAR_NAME ?= pd-sidecar
 IMAGE_REGISTRY ?= ghcr.io/llm-d
 IMAGE_TAG_BASE ?= $(IMAGE_REGISTRY)/$(PROJECT_NAME)
 EPP_TAG ?= dev
+export EPP_TAG
```

Comment:
What is this `export` for? 
if you want to share the same variable between Makefile and script, I recommend you to use env file. like kserve: https://github.com/kserve/kserve/blob/master/kserve-deps.env

Reply:
Please don't do that. It is much easier to work from the command line that to have to update all sorts of env files. Especially when testing all sorts of permutations of things.